### PR TITLE
Enable token and serviceUrl through Environment Variables

### PR DESCRIPTION
**Pull request by rikonor**

### CODE REVIEW

File: server/prerender.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-  Meteor.settings.PrerenderIO.token
-) {
-  prerenderToken = Meteor.settings.PrerenderIO.token;
-  prerenderio.set('prerenderToken', prerenderToken);
-  if (Meteor.settings.PrerenderIO.prerenderServiceUrl) {
-    prerenderServiceUrl = Meteor.settings.PrerenderIO.prerenderServiceUrl;
-    prerenderio.set('prerenderServiceUrl', prerenderServiceUrl);
-  }
-}
+(function() {
```
Comment (dferber90) [inline server/prerender.js]: Is there a reason this is wrapped in an IIFE?

File: server/prerender.js [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+  var prerenderToken = Meteor.settings.PrerenderIO && Meteor.settings.PrerenderIO.token;
+  prerenderToken = prerenderToken || process.env.PRERENDERIO_TOKEN; // Fallback to environment variable
+  if (prerenderToken) prerenderio.set('prerenderToken', prerenderToken);
+
+  // service url
+  var prerenderServiceUrl = Meteor.settings.PrerenderIO && Meteor.settings.PrerenderIO.prerenderServiceUrl;
+  prerenderServiceUrl = prerenderServiceUrl || process.env.PRERENDERIO_SERVICE_URL; // Fallback to environment variable
+  if (prerenderServiceUrl) prerenderio.set('prerenderServiceUrl', prerenderServiceUrl);
+
+  console.log("Setting up Prerender.io at " + prerenderServiceUrl  + " with token " + prerenderToken);
```
Comment (dferber90) [inline server/prerender.js]: I'd prefer the module to stay quiet by default

**Comment by dferber90** [comment]
Thanks for the PR @rikonor. Supporting environment variables as well is a great idea. They should probably even take precedence over the configuration in `Meteor.settings`.

However it has some issues with ESLint. Run `npm install` in the package's folder. Then, run `npm run lint` to see all issues. Please resolve them.

I just published a new version of this plugin which broke your PR. You'd have to pull the current `master` branch into yours, resolve the merge conflicts and push that new commit.

I'd accept the PR if you go through the steps here:
- make environment variables take precedence over `Meteor.settings`
- resolve ESLint issues
- possibly remove IIFE
- merge current `master` branch
- remove `console.log`, let module stay quiet by default

**Comment by rikonor** [comment]
Hey @dferber90,

Thank you for the feedback.
Please feel free to review the PR fixes based on your updated `master` and let me know if there's any other change you'd like.

**Comment by dferber90** [comment]
Thank you :) I've just released `v2.2.0_2` which contains your changes.

Let me know how it works for you! Haven't tested it myself.
This release also uses `prerender-node` version 2.2.0 instead of 2.1.0, which could be breaking some things as well.

**Pull request merged**